pymc3/tuning/scaling.py

```diff
@@ -78,7 +105,10 @@ def find_hessian_diag(point, vars=None, model=None):
 
 def guess_scaling(point, vars=None, model=None):
     model = modelcontext(model)
-    h = find_hessian_diag(point, vars, model=model)
+    try:
+        h = find_hessian_diag(point, vars, model=model)
+    except NotImplementedError:
```

Comment:
When you are using an individual Theano expression as the target distribution as in #804 and Theano can't automatically compute the second derivative of that expression. As for example when coding up the Stochastic Volatility target from the NUTS paper - it allows a first, but not a second derivative in Theano.